### DISCUSSION TITLE
How to use Popover with a custom anchor ref?

### DISCUSSION THREAD

**Discussion by kasvith** [radix-ui/primitives#2422] [Help]
Hi, I am developing a user mention system that is similar to screenshot attached.

For this, I need to open a popover and place it where the cursor is. I already know this element's reference(provided by Lexical editor). But I couldn't figure out how to use that `ref` given by editor to place item dynamically with radix-ui

Or is there a more suitable way to implement this?

<img width="242" alt="image" src="https://github.com/radix-ui/primitives/assets/13379595/723a8685-7e48-4c0c-8658-964a11b38457">

**Comment by benoitgrelard** [comment]
You could maybe use the `Anchor` part without the `Trigger` part, but I'm not sure on the impact on accessibility in such cases.
